server/database/models.py

```diff
@@ -2,19 +2,27 @@
 from flask_bcrypt import generate_password_hash, check_password_hash
 
 class Contact(db.Document):
-    name = db.StringField(required=True, unique=True)
-    email = db.EmailField(required=True, unique=True)
+    name = db.StringField(required=True)
+    email = db.EmailField(required=True)
     location = db.StringField(required=True)
-    date = db.DateField(required=True)
+    date = db.StringField(required=True)
     socials = db.ListField(required=True)
     notes = db.StringField(required=True)
+    groups = db.ListField(default=list)
```

Comment:
This is cool, i'll just need to add a reference field within the list field in the contacts variable so that one group can have many contacts. Same for the groups variable in the User class. it'll link multiple groups to one user.

Reply:
Ahhh sounds good. I did not know about that so good catch :)